packages/core/content-manager/server/src/controllers/collection-types.ts

```diff
@@ -35,7 +35,10 @@ const createDocument = async (ctx: any, opts?: Options) => {
   const sanitizeFn = async.pipe(pickPermittedFields, setCreator as any);
   const sanitizedBody = await sanitizeFn(body);
 
-  const { locale, status = 'draft' } = await getDocumentLocaleAndStatus(body);
+  const {
+    locale,
+    status = contentTypes.hasDraftAndPublish(strapi.getModel(model)) ? 'draft' : 'published',
```

Comment:
does that make sense that this goes into the getDocumentLocaleAndStatus? (Setting published or undefined by default if it doesnt have draft and publish)

Reply:
As you see fit :) 
My point is also, all the other controllers should set this value to published by default if D&P is disabled (maybe I am wrong?), as this applies to all methods of the document service model. Also, an undefined status should also work in this situation. (Document Service defaults to draft when necessary) so maybe we dont even need the default value